### PR TITLE
Add a pushed motd environment variable to VPN clients.

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description
This will push clients an environment variable with the URL to the motd page.  This page will contain the standard gov jibber-jabber as well as news about the COOL goings-on.  

There is a new required input to this module that will need to be configured by callers:
`client_motd_url = "https://github.com/cisagov/cool-system/blob/develop/motd.md#welcome-to-cisas-cloud-oriented-operations-lab-cool`

## 💭 Motivation and Context

Required control for ATO.

See: #24 

## 🧪 Testing

Directive tested in staging.  

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -25,6 +25,11 @@ variable "client_dns_server" {
   description = "The address of the DNS server to be pushed to the VPN clients."
 }
 
+variable "client_motd_url" {
```
Comment (jsf9k) [inline variables.tf]: Why is this a required variable?

**Review by jsf9k — APPROVED** [review]
:coffin:

File: README.md [hunk 2]
```diff
@@ -67,6 +67,7 @@ module "example" {
 | client_dns_search_domain | The DNS search domain to be pushed to VPN clients. | `string` | n/a | yes |
 | client_dns_server | The address of the DNS server to be pushed to the VPN clients. | `string` | n/a | yes |
 | client_inactive_timeout | The number of seconds of tolerable user inactivity before a client will be disconnected from the VPN. | `number` | `3600` | no |
+| client_motd_url | A URL to the motd page.  This will be pushed to VPN clients as an environment variable. | `string` | n/a | yes |
```
Comment (dav3r) [inline README.md]: Why is this variable required?  What if someone doesn't want to have a MOTD?  Can we safely default to an empty string or something else?

Comment (felddy) [reply README.md:70]: It is required because this is part of a required security control.

File: variables.tf (hunk 1 above)
Comment (felddy) [reply variables.tf:28]: It is required by our security team.  You are not allowed to not have a motd.

Comment (jsf9k) [reply variables.tf]: Right.  I guess the point is that maybe someone would want to use this TF module outside of the COOL and our security requirements.

Comment (felddy) [reply variables.tf]: Yup.  Took a second for me to think about this and realize this wasn't a `cool-*` repo.  Good catch all.

**Review by dav3r — APPROVED** [review]
Excellente! 👍

File: variables.tf (hunk 1 above)
Comment (jsf9k) [reply variables.tf]: The same thing goes for the timeout, although I assumed you could set that to `-1`.

Comment (felddy) [reply variables.tf]: The value would be `0`.  See inline docs:
https://github.com/cisagov/openvpn-server-tf-module/blob/2b1735823c35e93e06ab36ba6d30dbc4b85c06e5/cloudinit/openvpn-config.tpl.yml#L65-L66

**Pull request merged**